homepage/views.py

```diff
@@ -47,7 +48,11 @@ def courses(request):
 
 def reviews(request):
     reviews = Review.main_feed()
-    return render(request, 'homepage/reviews/reviews.html', {'reviews': reviews})
+    if(request.user.is_anonymous):
```

Comment:
```suggestion
    liked_reviews = [] if request.user.is_anonymous else User_Likes.get_liked_reviews_by_user(request.user)
```
WDYT?

Reply:
@oriAdler left it as it was, I think it's more readable